project/dexter-eclipse-ui/src/java/com/samsung/sec/dexter/eclipse/ui/OpenPlatzViewHandler.java

```diff
@@ -0,0 +1,36 @@
+package com.samsung.sec.dexter.eclipse.ui;
+
+import org.eclipse.core.commands.AbstractHandler;
+import org.eclipse.core.commands.ExecutionEvent;
+import org.eclipse.core.commands.ExecutionException;
+import org.eclipse.core.commands.IHandler;
+import org.eclipse.jface.dialogs.MessageDialog;
+import org.eclipse.ui.IViewPart;
+import org.eclipse.ui.IWorkbenchPart;
+import com.samsung.sec.dexter.core.config.DexterConfig;
+import com.samsung.sec.dexter.core.exception.DexterRuntimeException;
+import com.samsung.sec.dexter.eclipse.ui.util.EclipseUtil;
+import com.samsung.sec.dexter.eclipse.ui.view.PlatzView;
+
+public class OpenPlatzViewHandler extends AbstractHandler implements IHandler {
+
+	private IWorkbenchPart part;
+
+	@Override
+	public Object execute(ExecutionEvent event) throws ExecutionException {
+		try {
+			IViewPart platzPart = EclipseUtil.findView(PlatzView.ID);
+			final PlatzView platzview = (PlatzView) platzPart;
+			StringBuilder url = new StringBuilder();
+			url.append(DexterConfig.PLATZ_API_URL);
+			platzview.setUrl(url.toString());
+			EclipseUtil.showView(PlatzView.ID);
+
+		} catch (DexterRuntimeException e) {
+			MessageDialog.openError(part.getSite().getShell(), "PLATZ Error", "Cannot open the PLATZ View");
```

Comment:
In the eclipse documentation, if an exception occured, it should throw the ExecutionException.
 Please test which one is better between opening Message Dialog or throw ExecutionException.
 Read this: http://help.eclipse.org/juno/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fcore%2Fcommands%2FIHandler.html
 (If you can test it next version)